packages/react-server-cli/gulpfile.babel.js

```diff
@@ -1,8 +1,21 @@
 import eslint from "gulp-eslint";
 import gulp from "gulp";
 import babel from "gulp-babel";
+import jasmine from "gulp-jasmine";
 import logging from "react-server-gulp-module-tagger";
 
+function getSpecGlob (prefix) {
+	// add a wildcard onto the end if no file extension or wildcard
+	// currently present
+	let specGlob = "*[Ss]pec.js";
+	if (!specGlob.endsWith(".js") && !specGlob.endsWith("*")) {
```

Comment:
The other usage allows a CLI flag on the tests themselves to override `specs` (e.g. `gulp test --specs Foo`) and then appends a `*` to turn it into a glob prefix match if it isn't one already. I think here we could get rid of the `if`, unless we plan on supporting the CLI flag later.